src/IntegratedExperts/Behat/Relativity/Context/RelativityContextInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * @file
+ * Behat context interface to enable Relativity.
+ */
+
+namespace IntegratedExperts\Behat\Relativity\Context;
+
+use Behat\Behat\Context\Context;
+
+/**
+ * Interface RelativityContext.
+ */
+interface RelativityContextInterface extends Context
```

Comment:
rename to `RelativityAwareContext`. this will trigger PHPCS error - please see `behat-screenshot` project `phpcs.dst.xml` to know how it is excluded